web/public/js/services/user-utils.js

```diff
@@ -105,5 +105,22 @@ angular.module('cpZenPlatform').factory('userUtils', ['$location', '$window', '$
     return  avatar || overallDefault;
   }
 
+  userUtils.getTitleForUserTypes = function (userTypes) {
+    var title = [];
+    if (_.includes(userTypes, 'attendee-u13')) {
+      title.push($translate.instant('Ninja (Under 13)'));
```

Comment:
Actually, in addition to this, should we display "(Under 13)" and "(Over 13)" based on DoB instead of attendee-u13 and attendee-o13